src/Fabulous/Memo.fs

```diff
@@ -78,6 +79,7 @@ module Memo =
         {
             Key = MemoWidgetKey
             Name = "Memo"
+            TargetType = Unchecked.defaultof<_> // Memo isn't allowed in lists, so this will never get called
```

Comment:
I wonder if we can throw maybe in here? It is a getter right?

Reply:
It's not a getter. It's evaluated in the startup code, and so it would crash everytime